pkg/apis/aws/validation/shoot_test.go

```diff
@@ -48,6 +48,42 @@ var _ = Describe("Shoot validation", func() {
 		})
 	})
 
+	Describe("#ValidateNetworkingUpdate", func() {
```

Comment:
Does it make sense to have this as a table-driven test and check more combinations, e.g. IPv4 => IPv6,IPv4?